pkg/asset/installconfig/gcp/session.go

```diff
@@ -54,7 +54,6 @@ func loadCredentials(ctx context.Context) (*googleoauth.Credentials, error) {
 	for _, l := range loaders {
 		creds, err := l.Load(ctx)
 		if err != nil {
-			logrus.Debug(errors.Wrapf(err, "failed to load credentials from %s", l))
```

Comment:
@abhinavdahiya in addition to resolving your comments from yesterday, please note this new change. I removed this error message users would see this in terminal prompts, which I don't think is good ux:

`? Platform gcp
DEBUG failed to load credentials from loading from environment variable "GOOGLE_CREDENTIALS": invalid character '/' looking for beginning of value 
DEBUG failed to load credentials from loading from environment variable "GOOGLE_CLOUD_KEYFILE_JSON": empty environment variable 
DEBUG failed to load credentials from loading from environment variable "GCLOUD_KEYFILE_JSON": empty environment variable 
? Project ID openshift-dev-installer
? Region us-central1
DEBUG   Generating "Base Domain"...                
DEBUG failed to load credentials from loading from environment variable "GOOGLE_CREDENTIALS": invalid character '/' looking for beginning of value 
DEBUG failed to load credentials from loading from environment variable "GOOGLE_CLOUD_KEYFILE_JSON": empty environment variable 
DEBUG failed to load credentials from loading from environment variable "GCLOUD_KEYFILE_JSON": empty environment variable 
? Base Domain padillon.origin-gce.dev.openshift.com``
```